src/qt/rpcconsole.cpp

```diff
@@ -463,7 +463,19 @@ RPCConsole::RPCConsole(interfaces::Node& node, const PlatformStyle *_platformSty
     ui->dataDir->setToolTip(ui->dataDir->toolTip().arg(QString(nonbreaking_hyphen) + "datadir"));
     ui->blocksDir->setToolTip(ui->blocksDir->toolTip().arg(QString(nonbreaking_hyphen) + "blocksdir"));
     ui->openDebugLogfileButton->setToolTip(ui->openDebugLogfileButton->toolTip().arg(PACKAGE_NAME));
-    ui->peerConnectionTypeLabel->setToolTip(ui->peerConnectionTypeLabel->toolTip().arg("addnode").arg(QString(nonbreaking_hyphen) + "addnode").arg(QString(nonbreaking_hyphen) + "connect"));
+    ui->peerConnectionTypeLabel->setToolTip(
+        ui->peerConnectionTypeLabel->toolTip()
+            .arg(tr("<ul>"
+                    "<li>Inbound Relay: initiated by peer</li>"
+                    "<li>Outbound Full Relay: default</li>"
+                    "<li>Outbound Block Relay: does not relay transactions or addresses</li>"
+                    "<li>Outbound Manual: added using RPC %1 or %2/%3 configuration options</li>"
+                    "<li>Outbound Feeler: short-lived, for testing addresses</li>"
+                    "<li>Outbound Address Fetch: short-lived, for soliciting addresses</li>"
+                    "</ul>")
+                     .arg("addnode")
+                     .arg(QString(nonbreaking_hyphen) + "addnode")
+                     .arg(QString(nonbreaking_hyphen) + "connect")));
```

Comment:
How about
```diff
--- a/src/qt/rpcconsole.cpp
+++ b/src/qt/rpcconsole.cpp
@@ -463,19 +463,19 @@ RPCConsole::RPCConsole(interfaces::Node& node, const PlatformStyle *_platformSty
     ui->dataDir->setToolTip(ui->dataDir->toolTip().arg(QString(nonbreaking_hyphen) + "datadir"));
     ui->blocksDir->setToolTip(ui->blocksDir->toolTip().arg(QString(nonbreaking_hyphen) + "blocksdir"));
     ui->openDebugLogfileButton->setToolTip(ui->openDebugLogfileButton->toolTip().arg(PACKAGE_NAME));
-    ui->peerConnectionTypeLabel->setToolTip(
-        ui->peerConnectionTypeLabel->toolTip()
-            .arg(tr("<ul>"
-                    "<li>Inbound Relay: initiated by peer</li>"
-                    "<li>Outbound Full Relay: default</li>"
-                    "<li>Outbound Block Relay: does not relay transactions or addresses</li>"
-                    "<li>Outbound Manual: added using RPC %1 or %2/%3 configuration options</li>"
-                    "<li>Outbound Feeler: short-lived, for testing addresses</li>"
-                    "<li>Outbound Address Fetch: short-lived, for soliciting addresses</li>"
-                    "</ul>")
-                     .arg("addnode")
-                     .arg(QString(nonbreaking_hyphen) + "addnode")
-                     .arg(QString(nonbreaking_hyphen) + "connect")));
+
+    const std::vector<QString> list_items{
+        tr("Inbound Relay: initiated by peer"),
+        tr("Outbound Full Relay: default"),
+        tr("Outbound Block Relay: does not relay transactions or addresses"),
+        tr("Outbound Manual: added using RPC %1 or %2/%3 configuration options")
+            .arg("addnode")
+            .arg(QString(nonbreaking_hyphen) + "addnode")
+            .arg(QString(nonbreaking_hyphen) + "connect"),
+        tr("Outbound Feeler: short-lived, for testing addresses"),
+        tr("Outbound Address Fetch: short-lived, for soliciting addresses")};
+    const QString list = "<ul><li>" + Join(list_items, QString("</li><li>")) + "</li></ul>";
+    ui->peerConnectionTypeLabel->setToolTip(ui->peerConnectionTypeLabel->toolTip().arg(list));
 
     if (platformStyle->getImagesOnButtons()) {
         ui->openDebugLogfileButton->setIcon(platformStyle->SingleColorIcon(":/icons/export"));
```
?

Reply:
Nice, thanks! We were thinking in the same direction. Adapted it and added your name to the commit.

Reply:
Note, named it `CONNECTION_TYPE_DOC` to show up in git grepping with the constant of the same name in rpc/net.h